packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/index.mjml

```diff
@@ -5,7 +5,10 @@
 <mj-section>
   <mj-column>
     <mj-text css-class="text-header">
-      <span data-l10n-id="verifyLoginCode-title">Is this you signing in?</span>
+      <span data-l10n-id="verifyLoginCode-title-2" data-l10n-args="<%= JSON.stringify({ serviceName }) %>">Did you sign in to <%- locals.serviceName %>?</span>
```

Comment:
nit: "sign-in" not hyphenated here either (but is below on L11 in this file).

Reply:
same as above -- I think this is a verb vs noun use